### DISCUSSION TITLE
What specific model is used for finegrain-object-eraser demo?

### DISCUSSION THREAD

**Discussion by Sanster** [finegrain-ai/refiners#437] [Q&A] [answered]
Hi, I tried the demo at https://huggingface.co/spaces/finegrain/finegrain-object-eraser, and I think this is the best promptless generative object eraser model I have seen so far. I would like to ask which model this is? Thank you!

**Comment by dnsbrl** [comment, marked as answer]
Hey, thanks for your feedback! We trained our own model to make sure we meet our quality goals. We are thinking of open sourcing it fyi - stay tuned!
